distributed/deploy/cluster.py

```diff
@@ -427,10 +427,13 @@ def update():
         cluster_repr_interval = parse_timedelta(
             dask.config.get("distributed.deploy.cluster-repr-interval", default="ms")
         )
-        pc = PeriodicCallback(update, cluster_repr_interval * 1000)
-        self.periodic_callbacks["cluster-repr"] = pc
-        pc.start()
 
+        def install():
+            pc = PeriodicCallback(update, cluster_repr_interval * 1000)
+            self.periodic_callbacks["cluster-repr"] = pc
+            pc.start()
+
+        self.loop.add_callback(install)
```

Comment:
the PeriodicCallbacks in `self.periodic_callbacks` need to be started and stopped on the same loop as `self.loop` because they bind to `IOLoop.current()` when `PeriodicCallback.start` is called

When running a synchronous Cluster the loop will be running in a different thread to where `cluster._ipython_widget_` is called from